composer.json

```diff
@@ -18,12 +18,12 @@
     "require": {
         "php": ">=7.0.0",
         "swiftmailer/swiftmailer": "^6.0.1",
-        "symfony/dependency-injection": "~2.7|~3.4|~4.0",
+        "symfony/dependency-injection": "~2.7|~3.0|~4.0",
```

Comment:
I think using `3.3` could be better as versions before are not supported anymore.

Reply:
for all deps?